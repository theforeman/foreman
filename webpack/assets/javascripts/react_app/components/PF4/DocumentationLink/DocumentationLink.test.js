import { shallow } from 'enzyme';
import React from 'react';
import Link from './index';

describe('documentation links', () => {
  it('should have an external link to documentation', () => {
    const wrapper = shallow(<Link href="http://theforeman.org" />);

    expect(wrapper).toMatchSnapshot();
  });
});
