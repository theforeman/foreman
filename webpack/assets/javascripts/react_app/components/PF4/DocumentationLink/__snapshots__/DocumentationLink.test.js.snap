// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`documentation links should have an external link to documentation 1`] = `
<DropdownItem
  key="documentationUrl"
  onClick={[Function]}
  ouiaId="doc-url-dropdown"
>
  <QuestionCircleIcon />
   
  Documentation
</DropdownItem>
`;
